modules/control/README.md

```diff
@@ -3,6 +3,13 @@
 ## Introduction
 Based on the planning trajectory and the car's current status, the Control module uses different control algorithms to generate a comfortable driving experience. The Control module can work both in normal and navigation modes.
 
+Apollo 5.0 Control has following new features:
+ * **Unlock Low Speed / Reverse Driving Scenarios**: By introducing a series of control methodologies ("look-ahead" controller, etc.) and upgrading the control-oriented dynamic models, the control module significantly improves the control performance under low speed and reverse driving modes, so as to support the valet parking, pull-over and many other practical driving scenarios.
+ * **Reference/Feedback Signal Coordinate Transformation**: The coordinate of the planning trajectory ("reference" signal of control system) and the vehicle states ("feedback" signal of the control system) can be transformed between the center of the rear-axis and the center of mass of the vehicle, to develop the control performance with respect to various driving scenarios.
+ * **Optimized MPC Controller Algorithm**: The structure design of the MPC (Model Predictive Control) algorithm is optimized and consequently, the control accuracy with MPC controller, particularly at sharp turning scenarios, gets improved to a large extent.   
```

Comment:
The structural design of the MPC (Model Predictive Control) algorithm is optimized to greatly improve the control accuracy with the MPC controller, especially in scenarios that involve sharp turns.